examples/main.tf

```diff
@@ -166,41 +169,43 @@ resource "instaclustr_cluster" "example-elasticsearch" {
 }
 
 resource "instaclustr_cluster" "validKC" {
-    cluster_name = "testcluster"
-    node_size = "t3.medium-10-gp2"
-    data_centre = "US_WEST_2"
-    sla_tier = "NON_PRODUCTION"
-    cluster_network = "192.168.0.0/18"
-    private_network_cluster = false
-    pci_compliant_cluster = false
-    cluster_provider = {
-        name = "AWS_VPC"
-    }
-    rack_allocation = {
-        number_of_racks = 3
-        nodes_per_rack = 1
-    }
+  cluster_name = "testcluster"
+  node_size = "t3.medium-10-gp2"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
+  cluster_network = "192.168.0.0/18"
+  private_network_cluster = false
+  pci_compliant_cluster = false
+  cluster_provider = {
+    name = "AWS_VPC"
+  }
+  rack_allocation = {
+    number_of_racks = 3
+    nodes_per_rack = 1
+  }
 
-    bundle {
-        bundle = "KAFKA_CONNECT"
-        version = "2.3.1"
-        options = {
-            target_kafka_cluster_id = "${instaclustr_cluster.example_kafka.cluster_id}"
-            vpc_id = "SEPARATE_VPC"
-        }
+  bundle {
+    bundle = "KAFKA_CONNECT"
+    version = "2.3.1"
+    options = {
+      target_kafka_cluster_id = "${instaclustr_cluster.example_kafka.id}"
+      vpc_id = "SEPARATE_VPC"
     }
+  }
 }
 
-resource "instaclustr_kafka_user" "kafka_user_charlie" {
```

Comment:
Should retain the example for kafka user resource and data source

Reply:
added them back